openmdao/drivers/test/test_scipy_optimizer.py

```diff
@@ -281,7 +281,7 @@ def test_simple_array_comp_SLSQP_scaler_unity_eq(self):
         prob.driver.options['optimizer'] = 'SLSQP'
         prob.driver.add_desvar('x', lower=-50.0, upper=50.0, scaler=np.array([1.0, 1.0]))
 
-        prob.driver.add_objective('o', scaler=np.array([1.0, 1.0]))
+        prob.driver.add_objective('o', scaler=1.0)
```

Comment:
So, this is a different problem than how scipy is outputting stuff in py 2 and 3? I think in general we want to support both integer and array specification of the scaler/adder in the scipy optimizer, so I think we probably need to fix the problem rather than the test. If you don't want to tackle that, it is probably fine to leave these two test failing (as a reminder at least.)